CHANGED

```diff
@@ -1,5 +1,7 @@
-11.11.2018
+12.11.2018
  14_SD_UT.pm - added devices QUIGG GT-7000 |Novy Pureline 6830 kitchen hood|CAME Drehtor Antrieb (develop - user feedback required)
+ !! Module has been completely redesigned !! protocol 34 & 86 are developID's
```

Comment:
sorry, hier müsste noch 29 ergänzt werden und das Kommentar geht ja grundsätzlich in richtung SIGNALduino und nicht SD_UT.

Reply:
Ich denke nun alles zu erschlagen haben ;-)